packages/community/src/llm/bedrock/amazon/utils.ts

```diff
@@ -132,7 +132,9 @@ export const mapBaseToolsToAmazonTools = (tools?: BaseTool[]): Tool[] => {
     return {
       toolSpec: {
         ...options,
-        inputSchema: parameters,
+        inputSchema: {
```

Comment:
@parhammmm what's the impact of using other models?

Reply:
Amazon only has the Nova models, so it wont affect none-nova modals (other models are one folder above)